source/deploy/standalone/dev_ssl.md.erb

```diff
@@ -0,0 +1,845 @@
+---
+title: Using SSL with Passenger in development
+section: walkthroughs
+---
+<%
+locals.merge!(INTEGRATION_MODE_STANDALONE)
+globals.merge!(INTEGRATION_MODE_STANDALONE)
+content_for(:sidebar) do
+    render_partial("../sidebar.html", locals: locals.merge(
+        limit_choices: { "intmode" => [INTEGRATION_MODE_STANDALONE] }
+    ))
+end
+%>
+
+<h1>Using SSL with Passenger in development</h1>
+
+It is a good idea to have your dev env be as close to your production env as possible, without getting in your way. For example setting up ssl in your dev env helps you to fix mixed content warnings so there are no surprises when you move to staging or production, and doesn't train you to click through ssl warnings in your browser.
+
```

Comment:
This page would benefit from a high level explanation of the steps I'm about to undertake.

Also, maybe you should start with the passenger specific part, because as far as passenger is concerned it is basically a one-liner, and if you use your production cert together with a one-liner in the hosts file you'd already be done. This is not recommended for several reasons, of which you already name a few, so then the story can continue with "these steps for getting a self-signed cert"